### PR TITLE
Feature/code upd 8.8.6

### PR DESCRIPTION
**Pull request by mr-valters**
Update to D8.8.0 & use drupal/core-recommended

Config export w changed default texts to en

### CODE REVIEW

File: drupal/sync/config_ignore.settings.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  4: system.site
-  6: uhsg_service_provider_details.settings
-  8: 'xmlsitemap_engines.*'
-  10: 'google_tag.*'
+  - 'webform.webform.*'
+  - 'devel.*'
+  - system.site
+  - uhsg_service_provider_details.settings
+  - 'xmlsitemap_engines.*'
+  - 'google_tag.*'
```
Comment (tormi) [inline drupal/sync/config_ignore.settings.yml:7]: I might be using here new D8.8 `$settings['config_exclude_modules']` feature instead, CR: https://www.drupal.org/node/3079028

**Comment by tormi** [comment]
I suggest you to add `"zaporylie/composer-drupal-optimizations": "^1.1"` to `require-dev` section of `composer.json` for performance reasons.

**Comment by tormi** [comment]
Another handy setting for `composer.json`, sorts packages:

```
    "config": {
        "sort-packages": true,
    }
```

**Comment by tormi** [comment]
`CONFIG_SYNC_DIRECTORY` is deprecated, CR: https://www.drupal.org/node/3018145
Instances:
- https://github.com/UniversityofHelsinki/Opetustila-esittely/blob/94ec1fe60e86367c86e8e84f9c4f0c0d5d1d23e9/drupal/scripts/composer/ScriptHandler.php
- https://github.com/UniversityofHelsinki/Opetustila-esittely/blob/94ec1fe60e86367c86e8e84f9c4f0c0d5d1d23e9/drupal/web/sites/default/settings.php#L133

**Comment by tormi** [comment]
I would update Drush as well.

**Comment by tormi** [comment]
This CR is also something to consider: https://www.drupal.org/node/2538996 because `config_installer` package is recently deprecated, https://www.drupal.org/project/config_installer

**Comment by tormi** [comment]
Drush aliases are not updated to Drush 9/10 syntax, https://github.com/UniversityofHelsinki/Opetustila-esittely/tree/master/drupal/drush. See an example here: https://github.com/wunderio/drupal-project/tree/master/drush

**Comment by tormi** [comment]
Use `xdebug: false` instead of commentin out - it will increase the performance: https://github.com/UniversityofHelsinki/Opetustila-esittely/blob/feature/code-upd-8.8.6/drupal/.lando.yml#L9

**Comment by tormi** [comment]
This should not be needed: https://github.com/UniversityofHelsinki/Opetustila-esittely/blob/feature/code-upd-8.8.6/drupal/.lando.yml#L41

File: drupal/sync/config_ignore.settings.yml (hunk 1 above)
Comment (mr-valters) [reply drupal/sync/config_ignore.settings.yml:7]: So in `web/sites/default/settings.php` I should add following:
`$settings['config_exclude_modules'] = ['webform.webform', 'devel', 'xmlsitemap_engines', 'google_tag'];`

And from `drupal/sync/config_ignore.settings.yml` remove following:
  - 'webform.webform.*'
  - 'devel.*'
  - 'xmlsitemap_engines.*'
  - 'google_tag.*'

Comment (tormi) [reply drupal/sync/config_ignore.settings.yml:7]: `devel` only, other components needs to be present in remote environments. But `stage_file_proxy` and `update` are good candidates as well - all modules needed in local env only.

**Comment by tormi** [comment]
`extras` is deprecated, use `build_as_root` instead: https://github.com/UniversityofHelsinki/Opetustila-esittely/blob/feature/code-upd-8.8.6/drupal/.lando.yml#L33

See the Lando build steps docs here: https://docs.lando.dev/config/services.html#build-steps

**Pull request merged**